xbmc/platform/linux/powermanagement/LinuxPowerSyscall.cpp

```diff
@@ -14,13 +14,21 @@
 #include "UPowerSyscall.h"
 #endif // HAS_DBUS
 
+#if TARGET_WEBOS
+#include "LunaPowerManagement.h"
+#endif
+
 #include <functional>
 #include <list>
 #include <memory>
 #include <utility>
 
 IPowerSyscall* CLinuxPowerSyscall::CreateInstance()
 {
+#if TARGET_WEBOS
+  return new CLunaPowerManagement();
```

Comment:
Can you adjust the linuxpowersyscall registration instead in cplatformlinux?

It might even be nice to make a new cplatformwebos.